lib/main.dart

```diff
@@ -0,0 +1,111 @@
+import 'dart:math';
+import 'names.dart';
+import 'package:word_generator/word_generator.dart';
+
+
+void main() {
+  runTask1();
+  runTask2();
+  runTask3();
+  
+}
+
+void runTask1() {
+  print('------------------- Task 1 -------------------');
+final separator = List.filled(20, '=').join('');
+
+  // Створення списку numbers
+  List<int> numbers = List.generate(100, (_) => Random().nextInt(101));
+
+  print('Список numbers:');
+  print(numbers);
+
+
+  print(separator);
+
+  if (numbers.length >= 65) {
+    print('65-й елемент: ${numbers[64]}');
+  } else {
+    print('Список занадто короткий для доступу до 65-го елемента.');
+  }
+
+print(separator);
+  // Вставлення 1000000000 на 50-ту позицію
+  if (numbers.length >= 50) {
+    numbers.insert(49, 1000000000);
+    print('Число 1000000000 вставлено на 50-ту позицію');
+  } else {
+    print('Список занадто короткий для вставки на 50-ту позицію.');
+  }
+print(separator);
+
+  // Видалення всіх значень: 24, 45, 66, 88
+  List<int> valuesToRemove = [24, 45, 66, 88];
+  numbers.removeWhere((number) => valuesToRemove.contains(number));
+  print('Після видалення 24, 45, 66, 88:');
+  print(numbers);
+
+print(separator);
+  // Підрахунок суми всіх елементів, що діляться на 3 без залишку
+  int sumDivisibleBy3 = 0;
+  for (int i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 3 == 0) {
+      sumDivisibleBy3 += numbers[i];
+    }
+  }
+  print('Сума елементів, що діляться на 3: $sumDivisibleBy3');
+
+print(separator);
+  // Створення нового списку temp для парних чисел
+  List<int> temp = [];
+  for (var number in numbers) {
+    if (number % 2 == 0) {
+      temp.add(number);
+    }
+  }
+
+  print('Кількість парних чисел у temp: ${temp.length}');
+}
+
+void runTask2() {
+  print('------------------- Task 2 -------------------');
+
+  Set<String> uniqueNames1 = Set.from(ukrainianNames1);
```

Comment:
```suggestion
final uniqueNames1 = Set.from(ukrainianNames1);
final  uniqueNames2 = Set.from(ukrainianNames2);
```

It’s better to use final for variables you don’t plan to change. Please double-check and apply this throughout the code where appropriate